apps/nextjs-website/src/components/BannerLink.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+import { BannerLink as BannerLinkEC } from '@pagopa/pagopa-editorial-components';
+import { BannerLinkProps } from '@pagopa/pagopa-editorial-components/dist/components/BannerLink';
+import { Icon } from '@mui/material';
+import { BannerLinkSection } from '@/lib/fetch/types/PageSection';
+import { formatValidMuiIcon, isValidMuiIcon } from '@/components/Icons';
+
+const makeBannerLinkProps = ({
+  decoration,
+  ctaButtons,
+  ...rest
+}: BannerLinkSection): BannerLinkProps => ({
+  ...(decoration && {
+    decoration: {
+      url: decoration.url,
+      alt: decoration.alternativeText,
+    },
+  }),
+  ...(ctaButtons.length > 0 && {
+    ctaButtons: ctaButtons.map((ctaBtn) => ({
+      ...ctaBtn,
+      color: rest.theme === 'dark' ? 'negative' : 'primary',
+      ...(isValidMuiIcon(ctaBtn.icon) && {
+        startIcon: <Icon>{formatValidMuiIcon(ctaBtn.icon)}</Icon>,
+      }),
+    })),
+  }),
+  ...rest,
+});
```

Comment:
Due to a bug in the BannerLink component we are not currently rendering the `body` parameter as HTML as we would like.

[A task](https://pagopa.atlassian.net/browse/B2BP-354) has been created to address the issue.

Once the issue has been resolved, we'll render `body` as we should.